src/bsl/tree-sitter/codeModelFactory.ts

```
@@ -0,0 +1,366 @@
+    if (constructor) {
+        return constructor(node)
+    } else {
+        return undefined
+    }
+}
+
+export function createExpressionSymbol(node: Node | null): BaseExpressionSymbol | undefined {
+    if (!node) {
+        return undefined
+    }
+    const constructor = expressionConstructors[node.type]
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)